vedro_jira_failed_reporter/_failed_reporter_plugin.py

```diff
@@ -183,10 +185,9 @@ def on_scenario_failed(self, event: Union[ScenarioPassedEvent, ScenarioFailedEve
             self._reporting_language.ISSUE_CREATED.format(jira_server=self._jira_server, issue_key=result_issue.key)
         )
 
-        traceback = event.scenario_result._step_results[-1].exc_info.traceback
```

Comment:
we don't need this section (issues linking) anymore i suppose?

if we make one issue per file we don't have any separate issues related with this file.